src/pointer.cr

```diff
@@ -491,17 +491,19 @@ struct Pointer(T)
     Slice.new(self, size)
   end
 
-  # Returns a `Slice` from a given null-terminated pointer array, with an optional,
+  # Returns a `Slice` from a given, null-terminated, pointer array, with an optional,
   # though recommended, limit.
   #
   # ```
   # ptr = LibC.getgrnam("wheel").value.gr_mem
-  # slice = ptr.to_null_terminated_slice(LibC::NGROUP_MAX)
+  # slice = ptr.to_slice_null_terminated(LibC::NGROUP_MAX)
   # ```
   def to_slice_null_terminated(limit : Int)
     count = 0
     while self[count]
-      raise("Limit reached without NULL terminator.") if count > limit
+      if count > limit
```

Comment:
Shouldn't this be `count >= limit`?

Reply:
Depends if we consider the limit to be inclusive or exclusive. I would think that it would be inclusive. Is there a better terminology you can think of?

Reply:
I view 'limit' as to this point and no further. I have added documentation indicating this in the latest push.